src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/jpa/History.kt

```diff
@@ -39,6 +41,43 @@ class History(
   val questions: SortedSet<HistoricalQuestion> = sortedSetOf(),
 ) : Comparable<History> {
 
+  fun findQuestion(code: String, sequence: Int) = this.questions.firstOrNull { it.code == code && it.sequence == sequence }
```

Comment:
Do we need to consider the sequence? Isn't a question "equal" to another if their codes match?

Sorry, maybe I'm missing something fundamental here (still looking at this PR and the prior one)

Reply:
In NOMIS